Assets/Proto/Configs/agent.proto

```diff
@@ -1,20 +1,25 @@
 syntax = "proto3";
 
-package micromissiles;
+package configs;
 
-import "configs/proto/dynamic_config.proto";
-import "configs/proto/state.proto";
+import "Configs/dynamic_config.proto";
+import "Simulation/state.proto";
 
 // Interceptor type enumeration.
 enum InterceptorType {
-  MICROMISSILE = 0;
+  INVALID_INTERCEPTOR = 0;
   HYDRA_70 = 1;
+  MICROMISSILE = 2;
 }
 
 // Threat type enumeration.
 enum ThreatType {
-  QUADCOPTER = 0;
-  UCAV = 1;
+  INVALID_THREAT = 0;
+  QUADCOPTER = 1;
+  UCAV = 2;
+  BRAHMOS = 3;
+  ASCM = 4;
+  FATEH_110B = 5;
 }
```

Comment:
This enum is user-extendable after compilation, right? But I'm concerned that the Unity-side implementation will not support us adding additional Threats and Interceptors to the agent.proto enum.

Reply:
Not extendable because the Protobuf is compiled.

Reply:
Will the `oneof` construct allow users to add more threats at runtime from the files present? 
Also seems kinda painful to maintain cause every time we add a new threat or interceptor model gotta go update the enum..

Reply:
Will remove all enums and just refer to static configurations by filename.